packages/cozy-konnector-libs/libs/utils.js

```diff
@@ -0,0 +1,12 @@
+const cozyClient = require('./cozyclient')
+
+const fetchAll = async doctype => {
```

Comment:
You could use `cozy.client.data.findAll(doctype)` https://cozy.github.io/cozy-client-js/data-api.html